src/rdev.rs

```diff
@@ -158,7 +158,7 @@ pub enum Button {
 
 /// In order to manage different OS, the current EventType choices is a mix&match
 /// to account for all possible events.
-#[derive(Debug)]
+#[derive(Debug, Clone)]
```

Comment:
Why not copy here ?

Reply:
This should implement Copy, that was an oversight on my part.